euclidlib/photo/_le3_2pcf_wl.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from os import PathLike
+from typing import TYPE_CHECKING, Tuple, Union
+from numpy.typing import NDArray
+
+from euclidlib.photo import photo_data
+
+if TYPE_CHECKING:
+    from collections.abc import Sequence
+    from typing import Any, TypeAlias
+
+
+# type alias
+_DictKey: "TypeAlias" = Union[str, int, Tuple["_DictKey", ...]]
+
+def xi_tpcf(
+    path: str | PathLike[str],
+    *,
+    include: Sequence[tuple[Any, ...]] | None = None,
+    exclude: Sequence[tuple[Any, ...]] | None = None,
```

Comment:
As far as I can tell, we are not actually including or excluding individual combinations below.

Reply:
I removed these lines now.